projects/chaos/src/main.c

```diff
@@ -29,7 +31,9 @@
 
 static CANStorage s_can_storage;
 static CANRxHandler s_rx_handlers[CHAOS_NUM_RX_HANDLERS];
-static EmergencyFaultStorage s_emergency_storage;
+static EmergencyFaultStorage s_emergency_storage = {
+  .id = SOFT_TIMER_INVALID_TIMER,
```

Comment:
You don't have an init for this?

Reply:
Added. Previously I was using clear as an init but since that had 0 as the timer id that killed a soft_timer it shouldn't have.